README.md

```diff
@@ -5,22 +5,295 @@
 Realm is a mobile database that runs directly inside phones, tablets or wearables.
 This repository holds the source code for the Realm SDK for Flutter™ and Dart™.
 
-# Alpha
-
 **This project is in the Alpha stage, All API's might change without warning and no guarantees are given about stability. Do not use it in production.**
 
-The previous preview version of Realm Dart can be found on this branch: https://github.com/realm/realm-dart/tree/preview
+## Getting Started
+
+* Import Realm.
+    ```dart
+    import 'package:realm/realm.dart';
+    ```
+
+* Define a data model class `_Car`.
+    ```dart
+    @RealmModel()
+    class _Car {
+      late String make;
+
+      late String model;
+    
+      int? kilometers = 500;
+    }
+    ```
+
+* Generate Generates RealmObject class `Car` from data model class `_Car`.
+
+    ```
+    flutter run realm generate
+    ```
+
+* Open a Realm and add some objects.
+
+    ```dart
+    var config = Configuration([Car.schema]);
+    var realm = Realm(config);
+
+    var car = Car("Telsa", "Model Y", kilometers: 5);
```

Comment:
I think it will be nice to show how the generated constructor looks like before using it in the example.

Reply:
I think this is not a good place for that. The readme is our quick getting started guide. Such details are for elsewhere. This sample at the very beginning severs only one purpose show some code. A picture is worth a thousand words right. So sample code here is the same.